cmd/gctcli/commands.go

```diff
@@ -1231,7 +1231,7 @@ func getOrders(c *cli.Context) error {
 var getOrderCommand = cli.Command{
 	Name:      "getorder",
 	Usage:     "gets the specified order info",
-	ArgsUsage: "<exchange> <order_id>",
+	ArgsUsage: "<exchange> <order_id> <pair>",
```

Comment:
Something that might be beneficial to add, but is not blocking here, is to add the `asset_type` as a parameter. That way, at the wrapper, we'll be able to format currency pairs and potentially call different endpoints depending on the asset type received